src/Stryker.CLI/Stryker.CLI.UnitTest/ConfigFileTests.cs

```diff
@@ -70,19 +66,19 @@ public void StrykerCLI_WithConfigFile_ShouldStartStrykerWithConfigFileOptions(st
 
             mock.VerifyAll();
 
-            actualOptions.DevMode.ShouldBe(true);
-            actualOptions.AdditionalTimeoutMS.ShouldBe(9999);
-            actualOptions.LogOptions.LogLevel.ShouldBe(LogEventLevel.Verbose);
-            actualOptions.ProjectUnderTestNameFilter.ShouldBe("ExampleProject.csproj");
-            actualOptions.Reporters.ShouldHaveSingleItem();
-            actualOptions.Reporters.ShouldContain(Reporter.ConsoleReport);
-            actualOptions.ConcurrentTestrunners.ShouldBe(1);
-            actualOptions.Thresholds.Break.ShouldBe(20);
-            actualOptions.Thresholds.Low.ShouldBe(30);
-            actualOptions.Thresholds.High.ShouldBe(40);
-            actualOptions.FilePatterns.Count().ShouldBe(2);
-            actualOptions.FilePatterns.ShouldContain(filePattern);
-            actualOptions.Optimizations.ShouldBe(OptimizationFlags.CoverageBasedTest | OptimizationFlags.AbortTestOnKill);
+            actualInputs.AdditionalTimeoutMsInput.SuppliedInput.ShouldBe(9999);
+            actualInputs.VerbosityInput.SuppliedInput.ShouldBe("trace");
+            actualInputs.ProjectUnderTestNameInput.SuppliedInput.ShouldBe("ExampleProject.csproj");
+            actualInputs.ReportersInput.SuppliedInput.ShouldHaveSingleItem();
+            actualInputs.ReportersInput.SuppliedInput.ShouldContain(Reporter.Json.ToString());
+            actualInputs.ConcurrencyInput.SuppliedInput.ShouldBe(1);
+            actualInputs.ThresholdBreakInput.SuppliedInput.ShouldBe(20);
+            actualInputs.ThresholdLowInput.SuppliedInput.ShouldBe(30);
+            actualInputs.ThresholdHighInput.SuppliedInput.ShouldBe(40);
+            actualInputs.MutateInput.SuppliedInput.ShouldHaveSingleItem();
+            actualInputs.MutateInput.SuppliedInput.ShouldContain("!**/Test.cs{1..100}{200..300}");
+            actualInputs.CoverageAnalysisInput.SuppliedInput.ShouldBe("perTest");
+            actualInputs.DisableBailInput.SuppliedInput.ShouldBe(true);
```

Comment:
Instead of all these comparisons, have you considered overriding `Equals` and `GetHashCode` on `StrykerInputs` to be able to just use `Assert.True(expectedInputs.Equals(actualInputs));` here?